hoodie-common/pom.xml

```diff
@@ -70,6 +70,18 @@
   </build>
 
   <dependencies>
+    <dependency>
```

Comment:
can we avoid pulling this into common? utilities should all either go to hoodie-utilities or exposed via hoodie-cli?